.gitignore

```diff
@@ -24,6 +24,9 @@ out/
 .gradle/
 build/
 
+# Idea files
+.idea
```

Comment:
Hello @JorgeGreek all idea files are unnecessary?

Reply:
They are not unnecessary but I understood that they do not need to be committed